api/src/main/java/org/apache/iceberg/actions/DeleteOrphanFiles.java

```diff
@@ -80,6 +81,41 @@ public interface DeleteOrphanFiles extends Action<DeleteOrphanFiles, DeleteOrpha
    */
   DeleteOrphanFiles executeDeleteWith(ExecutorService executorService);
 
+  /**
+   * Pass a mode for handling the files that cannot be determined if they are orphan
+   * The allowed modes are "IGNORE", "ERROR", "DELETE"
+   * Default prefixMismatchMode is "ERROR".
+   *
+   * @param mode mode for handling files that cannot be determined if they are orphan
+   * @return this for method chaining
+   */
+  default DeleteOrphanFiles prefixMismatchMode(PrefixMismatchMode mode) {
+    throw new UnsupportedOperationException(this.getClass().getName() + " does not implement prefixMismatchMode");
+  }
+
+  /**
```

Comment:
The doc here should be updated to reflect that we support commas in keys now.

```
/**
 * Passes schemes that should be considered equal.
 * <p>
 * The key may include a comma-separated list of schemes. For instance, Map("s3a,s3,s3n", "s3").
 *
 * @param equalSchemes list of equal schemes
 * @return this for method chaining
 */
```

Reply:
done